lib/services/pcmk-dbus.h

```diff
@@ -14,19 +14,32 @@
 #    define DBUS_TIMEOUT_USE_DEFAULT -1
 #  endif
 
+G_GNUC_INTERNAL
```

Comment:
Do we define G_GNUC_INTERNAL anywhere if we're not using gcc? Perhaps we should define our own symbol, that evaluates to G_GNUC_INTERNAL if gcc is in use, and empty otherwise.